browser/src/control/Control.Zotero.js

```diff
@@ -426,6 +457,55 @@ L.Control.Zotero = L.Control.extend({
 		});
 	},
 
+	enableDisableNextPrevious: function(hide) {
+		this.map.fire('jsdialogupdate', {
+			data: {
+				jsontype: 'dialog',
+				action: 'update',
+				id: 'ZoteroDialog',
+				control: {
```

Comment:
can we create helper function: `updateControl(controlJSON)` ?
all the time we repeat few lines, you need to pass only internal "control"

Reply:
(can be used in many places in this file)